src/modes/shop/shop.cpp

```diff
@@ -1902,92 +1902,100 @@ void ShopMode::SetPriceLevels(SHOP_PRICE_LEVEL buy_level,
     _sell_price_level = sell_level;
 }
 
-void ShopMode::AddItem(uint32_t object_id, uint32_t stock)
+bool ShopMode::AddItem(uint32_t object_id, uint32_t stock)
 {
     if(IsInitialized()) {
         IF_PRINT_WARNING(SHOP_DEBUG) << "Function called after shop was already initialized."
                                      << std::endl;
-        return;
+        return false;
     }
 
     if(object_id == private_global::OBJECT_ID_INVALID ||
-           object_id >= private_global::OBJECT_ID_EXCEEDS) {
+       object_id >= private_global::OBJECT_ID_EXCEEDS) {
```

Comment:
Np, it's just the full codebase is that way and I found it useful to avoid (my own) mistakes. ;)